content/using/_index.md

```diff
@@ -31,3 +31,11 @@ application.
 Urbit can be run in a variety of contexts. Our [Getting Started
 Guide](@/getting-started.md) covers how to run an Urbit locally on your
 computer; this section covers alternative ways of running Urbit.
+
+### [Urbit on Raspberry Pi](@/using/raspberrypi/_index.md)
+
+This How-To guide will walk you through setting up your Urbit planet on
+a Raspberry Pi 4 in a cooling case with an optimally configured SATA SSD connected 
+via the USB 3 port. Urbit will run great, and you’ll have enough integrated
+high-speed storage to run other Urbit-related applications on the same device
+in the future if you wish (e.g. a Bitcoin full node, Lightning node, etc).
```

Comment:
I'd like to have this within the **Running Urbit** nav tree instead of having it as a top-level folder.